reactor-core/src/test/java/reactor/core/CurrentContextArchTest.java

```diff
@@ -36,6 +36,7 @@
 			.withImportOption(ImportOption.Predefined.DO_NOT_INCLUDE_JARS)
 			.importPackagesOf(CoreSubscriber.class);
 
+	// This is ok as this class tests the deprecated FluxProcessor. Will be removed with it in 3.5.
```

Comment:
The current comment has the benefit of explaining that this will be removed as a side effect of `FluxProcessor` being removed. Which is really the (small) value add of this PR when you search for `SuppressWarning("deprecation")`. Simplifying the texts too much removes context. We can always search for `removed.*3\.5` if needed, wdyt?

Reply:
yeah ok. my initial thought was that going the extra mile would make the search for things to remove trivial in 3.5.0, but it is probably safer to assume no consistent message and cast a wider net the day of the removal (since it will be easy to forget about the "right" message in future deprecations)

Reply:
note that we also have related issues to track what has to be removed in 3.5 per topic (#2478, #2207, #2204 ...)